scripts/databuilder/foundry_engine.py

```diff
@@ -5,7 +5,7 @@
 from hashlib import blake2b
 import json
 
-DEFAULT_POKEMON_MANEUVERS = ['Struggle', 'Struggle - Throw' 'Grapple', 'Help Another', 'Cover An Ally', 'Run Away', 'Ambush', 'Clash', 'Evasion', 'Stabilize An Ally']
+DEFAULT_POKEMON_MANEUVERS = ['Struggle - Physical', 'Struggle - Special', 'Grapple', 'Help Another', 'Cover An Ally', 'Run Away', 'Ambush', 'Clash', 'Evasion', 'Stabilize An Ally']
```

Comment:
Did you need to do anything with Struggle - Throw which also exists, and if Struggle is getting duplicated we shouldn't keep the combined one